packages.html

```diff
@@ -6,8 +6,8 @@
 	<input id="searchbox" id="obs-q" type="text" name="q" value="" class="form-control w-auto mx-auto mb-2 mr-md-2" lang="en" placeholder="Search for package"/>
 	<select name="baseproject"  class="custom-select mx-auto mb-2 mr-md-2" onchange='changeAction(this.value)'>
 		<option value="openSUSE:Factory">Tumbleweed</option>
-		<option value="openSUSE:15.2">Leap 15.2</option>
 		<option value="openSUSE:15.3">Leap 15.3</option>
+		<option value="openSUSE:15.4">Leap 15.4</option>
```

Comment:
Would you work on a feature to automatically update this based on https://get.opensuse.org/api/v0/distributions.json?

Reply:
Tbh, I don't know Ruby at all :(

But I'm going to study it, as there are plans to fix other components.

Reply:
I suggest manual adjustments for now, so that beginners aren't confused by this fine roughness.

Then, in another pull request, add the ability to take repository names from the API.